client/src/app/users/user.service.spec.ts

```diff
@@ -1,9 +1,23 @@
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpParams } from '@angular/common/http';
 import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
-import { TestBed } from '@angular/core/testing';
+import { TestBed, waitForAsync } from '@angular/core/testing';
+import { defer, of } from 'rxjs';
 import { User } from './user';
 import { UserService } from './user.service';
 
+/**
+ * Create async observable that emits-once and completes
+ * after a JS engine turn.
+ */
+// This was lifted from https://stackoverflow.com/a/52959047
+// Not sure why ESLint is grumpy about it, but it should be moved
+// to some sort of utility location since it would presumably
```

Comment:
Any thoughts on where this should go? We don't really have a "utility" area at the moment. Should we make one?

Reply:
With the suggestion from @helloworld12321 to use `of()`, this part can go away.